src/topology/continuous_function/algebra.lean

```diff
@@ -113,6 +113,28 @@ instance {α : Type*} {β : Type*} [topological_space α] [topological_space β]
   ..continuous_map.semigroup,
   ..continuous_map.has_one }
 
+@[to_additive]
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {β : Type*} [monoid β] [topological_space β]
+  [has_continuous_mul β] : has_continuous_mul C(α, β) :=
+⟨begin
+  refine continuous_of_continuous_uncurry _ _,
+  change continuous ((λ (x : β × β), x.1 * x.2) ∘
+    (λ (x : (C(α, β) × C(α, β)) × α), (x.1.1 x.2, x.1.2 x.2))),
+  have h1 : continuous ((λ (x : C(α, β) × α), x.1 x.2) ∘
+      (λ (x : (C(α, β) × C(α, β)) × α), (x.1.1, x.2))) :=
+  continuous.comp continuous_ev ((continuous_fst.comp continuous_fst).prod_mk continuous_snd),
+  have h2 : continuous ((λ (x : C(α, β) × α), x.1 x.2) ∘ (λ (x : (C(α, β) × C(α, β)) × α),
+    (x.1.2, x.2))) := continuous.comp continuous_ev
+      ((continuous_snd.comp continuous_fst).prod_mk continuous_snd),
+  refine (continuous_fst.comp (continuous.prod_mk _ _)).mul
+    (continuous_snd.comp (continuous.prod_mk _ _)),
+  { convert h1, },
```

Comment:
You can get rid of the need for these `convert`s by stating the desired version of `h1` originally, rather than something which is syntactically different.  Like
```lean
  have h1 : continuous (λ x : (C(α, β) × C(α, β)) × α, x.1.1 x.2) :=
```
Please make this change here and in the other proof.

Reply:
Done